plugins/polio/js/src/domains/Notifications/Table/NotificationsTable.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { FunctionComponent } from 'react';
+import { Column, useSafeIntl } from 'bluesquare-components';
+
+import { DateCell } from '../../../../../../../hat/assets/js/apps/Iaso/components/Cells/DateTimeCell';
+import { TableWithDeepLink } from '../../../../../../../hat/assets/js/apps/Iaso/components/tables/TableWithDeepLink';
+import { handleTableDeepLink } from '../../../../../../../hat/assets/js/apps/Iaso/utils/table';
+
+import MESSAGES from '../messages';
+import { DeleteNotificationModal } from '../Modals/NotificationsDeleteModal';
+import { EditNotificationModal } from '../Modals/NotificationsCreateEditModal';
+import { NOTIFICATIONS_BASE_URL } from '../../../constants/routes';
+import { useGetNotifications } from '../hooks/api';
+import {
+    ApiNotificationsParams,
+    DropdownsContent,
+    NotificationsParams,
+} from '../types';
+
+type Props = { params: NotificationsParams; dropdownContent: DropdownsContent };
```

Comment:
I think you need to either make `dropDownContent`optional: 
```suggestion
type Props = { params: NotificationsParams; dropdownContent?: DropdownsContent };
```

Or cast the type when you pass the props (see suggestion in index.tsx)